ingestion/src/lib.rs

```diff
@@ -53,7 +53,14 @@ pub type PgConnectionPool = bb8::Pool<PostgresConnectionManager<NoTls>>;
 
 pub type PooledPgConn<'a> = PooledConnection<'a, PostgresConnectionManager<NoTls>>;
 
-type ParamConversions = Arc<HashMap<String, (String, i32)>>;
+#[derive(Clone, Debug)]
+pub struct Param {
```

Comment:
I already raised this when this was used as a param, but I think this could be named more clearly